File: BE/src/main/java/com/example/hotel/model/entity/Blog.java
package com.example.hotel.model.entity;

import com.example.hotel.model.enums.BookingStatus;
import com.fasterxml.jackson.annotation.JsonManagedReference;
import lombok.*;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class Blog {
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private Long id;

    private String title;

    @Lob
    private String description;

    @Lob
    private String shortDescription;

    @JsonManagedReference
    @OneToOne(mappedBy = "blog", cascade = CascadeType.ALL)
    private Image image;

    private String author;

    @CreationTimestamp
    private LocalDateTime createdTime;

    @UpdateTimestamp
    private LocalDateTime updatedTime;
}

File: FE-ADMIN/demo/src/app/core/model/blog.ts
export class Blog {
  id: number;
  title: string;
  image?: string;
  description: string;
  shortDescription: string
  author: string;

  constructor() {
    this.id = 0
    this.title = '';
    this.image = '';
    this.description=  '';
    this.shortDescription= '';
    this.author=  '';
  }

}
